src/Client/Factory/AbstractFactory.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Werkspot\KvkApi\Client\Builder;
+namespace Werkspot\KvkApi\Client\Factory;
 
-abstract class AbstractBuilder
+abstract class AbstractFactory
```

Comment:
the `Factory` name becomes a better fit since we were not working the `Builder` pattern.